src/Config/ProjectsConfiguration.php

```diff
@@ -55,7 +55,7 @@ public function getConfigTreeBuilder()
                                         ->arrayNode('php')->prototype('scalar')->defaultValue([])->end()->end()
                                         ->arrayNode('services')->prototype('scalar')->defaultValue([])->end()->end()
                                         ->scalarNode('target_php')->defaultNull()->end()
-                                        ->scalarNode('phpunit_version')->defaultValue('8')->end()
+                                        ->scalarNode('phpunit_version')->defaultValue('8.5.0')->end()
```

Comment:
Shouldn't this be removed entirely? :thinking:

Reply:
Done.